libcontainer/nsenter/cloned_binary.c

```diff
@@ -65,19 +65,20 @@
 #  else
 /* These values come from <https://fedora.juszkiewicz.com.pl/syscalls.html>. */
 #    warning "libc is outdated -- using hard-coded SYS_memfd_create"
-#    if defined(__x86_64__) // x86_64
+#    if defined(__x86_64__)	// x86_64
 #      define SYS_memfd_create 319
-#    elif defined(__i386__) // i386
+#    elif defined(__i386__)	// i386
 #      define SYS_memfd_create 356
-#    elif defined(__ia64__) // ia64
+#    elif defined(__ia64__)	// ia64
 #      define SYS_memfd_create 1340
-#    elif defined(__arm__) // arm
+#    elif defined(__arm__)	// arm
 #      define SYS_memfd_create 385
-#    elif defined(__aarch64__) // arm64
+#    elif defined(__aarch64__)	// arm64
 #      define SYS_memfd_create 279
-#    elif defined(__ppc__) || defined(__ppc64__) // ppc + ppc64
+#    elif defined(__ppc__) || defined(__ppc64__)// ppc + ppc64
 #      define SYS_memfd_create 360
-#    elif defined(__s390__) || defined(__s390x__) // s390(x)
+#    elif defined(__s390__) || defined(__s390x__)
+						// s390(x)
```

Comment:
This one doesn't look right? I'd expected the old one here (also notice that it's using tabs to indent the _comment_, whereas lines above use spaces for indentation (but tabs for the last comments? (I'll leave a separate comment about that)

<img width="518" alt="Screenshot 2021-03-18 at 10 00 24" src="https://user-images.githubusercontent.com/1804568/111599927-209ce580-87d1-11eb-9526-131f9ff249e4.png">

So there's _six_ tabs in front of the comment; is it trying to fit it into columns?

Anticipating something like;

- line 78, being _slightly_ shorter; comment starts at position 49 (so don't add tabs to align)
- line 80: comment started at position 50/51; 
    - -> "just beyond the cut"
    - -> add 6 tabs (24 positions?) to put it in the next column
    - -> now line-width is at 50+24+10 (comment-length)
    - -> which is > 80 characters, so wrap the line


That said, the comment at line 376 did _not_ get reformatted/wrapped, and is at (assuming tab=4) 100 positions wide, so not sure.

Reply:
I don't like it either, but this is the way indent works (trying to bring the adjacent comments to the same column, and, I suspect, have some predefined preferred column position for such one-line comments).

Overall it makes sense to me -- this is how it looks like for me:

![Screenshot from 2021-03-18 16-43-41](https://user-images.githubusercontent.com/4522509/111711566-4514b400-8809-11eb-94d7-633fdf2de6f8.png)

(note that ppc and and s390 are aligned -- I suspect they are not for you as you have tabstop != 8)

Reply:
BTW one way to fix this is remove those comments (they are kinda redundant anyway)

Reply:
Removed `// $ARCH` comments